openapi/workflow_execution_service.swagger.yaml

```diff
@@ -456,6 +456,7 @@ definitions:
       - EXECUTOR_ERROR
       - SYSTEM_ERROR
       - CANCELED
+      - CANCELING
```

Comment:
I changed it to a single L to match "CANCELED". (Both spellings are okay for both words, so let's be consistent.)